ninja

```diff
@@ -0,0 +1 @@
+/usr/bin/ninja
```

Comment:
Did you mean to check this in?

Reply:
@manthey opps, no it must've been added automatically and escaped my attention. I removed it now.